scripts/github/deploy.sh

```diff
@@ -67,14 +67,19 @@ echo "Updating version from ${PREVIOUS_VERSION} to ${NEW_VERSION}"
 # Update package files versions, project inter-dependencies and lerna.json with new version number
 lerna version "${NEW_VERSION}" --yes --no-push --force-publish --exact
 
-# Update package lock with new version number
-lerna exec -- npm install --package-lock-only
-
 # Generate changelog information for changes since the last tag
 echo "Generating changelog updates for all changes between ${PREVIOUS_VERSION} and ${NEW_VERSION}"
 lerna-changelog --from "${PREVIOUS_VERSION}" --to "${NEW_VERSION}" | cat - CHANGELOG.md > CHANGELOG.new && mv CHANGELOG.new CHANGELOG.md
 git commit -a --amend --no-edit --no-verify
 
+# Publish packages to npm
+echo "Publishing latest packages to npm"
+lerna publish --registry=https://registry.npmjs.org/ from-git --yes --pre-dist-tag rc --no-verify-access
+
+# Update package lock with new version number
+echo "Updating package-lock.json for all packages"
+lerna exec -- npm install --package-lock-only
+
```

Comment:
I'm not sure about this - we'd be publishing before updating package locks, so the published versions wouldn't have the latest packages. I think we need to hold off merging until we've done some more on-branch testing